symengine/matrix.h

```diff
@@ -374,6 +383,12 @@ class CSRMatrix : public MatrixBase
         RCP<const Basic> (&bin_op)(const RCP<const Basic> &,
                                    const RCP<const Basic> &));
 
+    std::tuple<std::vector<unsigned>, std::vector<unsigned>, vec_basic>
+    as_vectors() const
+    {
+        return std::make_tuple(p_, j_, x_);
```

Comment:
I'm not sure if `std::make_tuple` will do the correct thing since `p_, j_, x_` are const objects. Maybe this is the reason for the appveyor failure

Reply:
Oh, I didn't think of that. Nice catch. I'll copy construct some locals and std::move them. I'm firing up a RDP build on appveyor in parallel.

Reply:
Same problem still. No pop-up in the RDP server. Will try to reproduce the error locally in a Windows VM.